packages/client/src/stores/ConfigEditorStore.ts

```diff
@@ -96,8 +96,12 @@ export default class ConfigEditorStore {
 
   @action.bound
   setCanaryConfigObject(canaryConfigObject: CanaryConfig): void {
-    this.canaryConfigObject = canaryConfigObject;
-    canaryConfigObject.metrics.forEach(metric => metric.groups.forEach(g => this.createNewGroup(g, false)));
+    if (canaryConfigObject.metrics == null) {
+      this.canaryConfigObject = canaryConfigObject.canaryConfig
```

Comment:
`canaryConfig` does not exist on `canaryConfigObject` (see [here](https://github.com/Nike-Inc/referee/blob/master/packages/client/src/domain/Kayenta.ts#L118:L132)).

What is the purpose of this conditional statement?

Reply:
So we post to the `/standalone_canary_analysis` endpoint of our standalone Kayenta API to have it start monitoring and for that our json needed to be structured with our configuration in an object called `canaryConfig`, but it seems like referee expects the config to be flat, so this was sort of a quick fix to make that work. What would be the best way to make the configuration work for both?